easybuild/easyconfigs/r/RELION/RELION-2.0.1-foss-2016a-CUDA-7.5.18.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.0.1'
+commit_id = 'a6225608e270'
```

Comment:
same question as in #3683, let's figure it out there and then port whatever we come up with in here